src/domains/event/dto/index.ts

```diff
@@ -84,7 +84,7 @@ export class IssueChangeLogDTO {
 
 export class BlockerStatusModificationDTO {
```

Comment:
If it happens that you find something not documented, please, try to add it.

Reply:
done